get-started/07_multi_container.md

```diff
@@ -165,21 +161,12 @@ The todo app supports the setting of a few environment variables to specify MySQ
 - `MYSQL_PASSWORD` - the password to use for the connection
 - `MYSQL_DB` - the database to use once connected
 
->**warning** 
->Setting Connection Settings via Env Vars
->While using env vars to set connection settings is generally ok for development, it is **HIGHLY DISCOURAGED**
->when running applications in production. Diogo Monica, the former lead of security at Docker, 
->[wrote a fantastic blog post](https://diogomonica.com/2017/03/27/why-you-shouldnt-use-env-variables-for-secret-data/)
->explaining why. 
->  
->A more secure mechanism is to use the secret support provided by your container orchestration framework. In most cases,
->these secrets are mounted as files in the running container. You'll see many apps (including the MySQL image and the todo app)
->also support env vars with a `_FILE` suffix to point to a file containing the variable. 
->   
->As an example, setting the `MYSQL_PASSWORD_FILE` var will cause the app to use the contents of the referenced file 
->as the connection password. Docker doesn't do anything to support these env vars. Your app will need to know to look for
->the variable and get the file contents.
-
+>**Warning**
+>
+>**Setting Connection Settings via Env Vars**
+>
+>While using env vars to set connection settings is generally ok for development, it is **HIGHLY DISCOURAGED** when running applications in production. Diogo Monica, the former lead of security at Docker, [wrote a fantastic blog post](https://diogomonica.com/2017/03/27/why-you-shouldnt-use-env-variables-for-secret-data/){:target="_blank" rel="noopener" class="_"} explaining why. A more secure mechanism is to use the secret support provided by your container orchestration framework. In most cases, these secrets are mounted as files in the running container. You'll see many apps (including the MySQL image and the todo app) also support env vars with a `_FILE` suffix to point to a file containing the variable. As an example, setting the `MYSQL_PASSWORD_FILE` var will cause the app to use the contents of the referenced file as the connection password. Docker doesn't do anything to support these env vars. Your app will need to know to look for the variable and get the file contents.
+{: .warning}
```

Comment:
Perhaps combine the "warning" with the next heading; something like

```markdown
> **Warning on using environment variables**
```

Looks like you un-wrapped this paragraph; for consistency, might want to add a space after each `>` as well;

```suggestion
> **Warning on using environment variables**
>
>
> While using env vars to set connection settings is generally ok for development,
> it is **HIGHLY DISCOURAGED** when running applications in production. Diogo Monica,
> the former lead of security at Docker, [wrote a fantastic blog post](https://diogomonica.com/2017/03/27/why-you-shouldnt-use-env-variables-for-secret-data/){:target="_blank" rel="noopener" class="_"}
> explaining why. A more secure mechanism is to use the secret support provided
> by your container orchestration framework. In most cases, these secrets are
> mounted as files in the running container. You'll see many apps (including the
> MySQL image and the todo app) also support env vars with a `_FILE` suffix to
> point to a file containing the variable. As an example, setting the
> `MYSQL_PASSWORD_FILE` var will cause the app to use the contents of the referenced
> file as the connection password. Docker doesn't do anything to support these
> env vars. Your app will need to know to look for the variable and get the file
> contents.
{: .warning}
```

Reply:
Aah, I thought we've used the unwrapped version for Notes and other admonitions. The output looks similar though. Any specific reason for wrapping these?

I'll add a space after `>`

Reply:
Mostly the same as for other content; the wrapping helps with a) readability when reading the source, and b) reviews (being able to comment / make changes on a specific line)